windows/jitsi-meet/render.js

```diff
@@ -0,0 +1,94 @@
+const http = require('../../modules/httpcontrol');
+const config = require('../../config.js');
+const listeningPort = process.env.PORT || 8000;
+
+/**
+ * The jitsi-meet api.
+ */
+let api;
+
+window.onload = function() {
+    http.init(listeningPort);
+    http.onReceivedCommand((type, args) => {
+        try {
+            const message = handleCommand(type, args);
+
+            http.sendResponse(true, message);
+        } catch (error) {
+            http.sendResponse(false, error.message);
+        }
+    });
+};
+
+/**
+ * Creates the iframe that will load Jitsi Meet.
+ *
+ * @param {string} room - conference room name
+ * @returns {null}
+ */
+function createJitsiIframe(room) {
+    const options = {
+        roomName: room,
+        width: 800,
+        height: 600,
+        parentNode: document.querySelector('#meet')
+    };
+
+    api = new JitsiMeetExternalAPI(config.JITSI_MEET_DOMAIN, options);
```

Comment:
you need to add listener to know when the conference was left so you can dispose the api object.